internal/runbits/requirements/requirements.go

```diff
@@ -174,10 +180,8 @@ func (r *RequirementOperation) ExecuteRequirementOperation(requirementName, requ
 			}
 			return locale.WrapInputError(err, "package_ingredient_alternatives", "", requirementName, strings.Join(suggestions, "\n"))
 		}
-		if name := packages[0].Ingredient.Name; name != nil && requirementName != *name {
-			logging.Debug("Requirement to install's letter case differs from Platform's ('%s' != '%s')", requirementName, *name)
-			requirementName = *name // match case
-		}
+
+		origRequirementName, requirementName = requirementName, normalized
```

Comment:
Why does origRequirementName need to be assigned again?

Reply:
We need to keep track of what the user requested to be installed, and what was actually installed. That way we can notify the user that something was silently changed on them.

Reply:
The value doesn't seem to be different than when set at line 160.